domain-cc/cc-api-controller/src/main/java/gov/va/vro/api/cc/v3/CCResource.java

```diff
@@ -0,0 +1,25 @@
+package gov.va.vro.api.cc.v3;
```

Comment:
I'm not sure why xample is at v3, but maybe you want to start at v1.

Reply:
fixed

Reply:
@lukey-luke You did not misinterpret. I've been using the version numbers in the Java package name to convey the VRO API version ([v1 and v2](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Roadmap#application-development-tasks)) for simplicity. Before the CC Team, there was only 1 API to deal with, so any changes to the API would increment the version.

Let's keep it `v3` for now so there's no confusion with VRO API v1.